packages/routing-utils/src/schemas.ts

```diff
@@ -207,16 +233,21 @@ export const redirectsSchema = {
 export const headersSchema = {
   type: 'array',
   maxItems: 1024,
+  description: 'A list of header definitions.',
   items: {
     type: 'object',
     additionalProperties: false,
     required: ['source', 'headers'],
     properties: {
       source: {
+        description:
+          'A pattern that matches each incoming pathname (excluding querystring)',
         type: 'string',
         maxLength: 4096,
       },
       headers: {
+        description:
+          'An array of key/value pairs representing each response header.',
         type: 'array',
         maxItems: 1024,
         items: {
```

Comment:
Missing description on `key` and `value` below

Reply:
Copied from here: https://github.com/vercel/api/blob/4ec9ff002e8d866754ace4aaee8c74da3c9e010d/utils/deployments-util/vercel-json-schema.ts#L237